modules/kubernetes-addons/crossplane/aws-provider/aws-provider.yaml

```
@@ -4,6 +4,6 @@ kind: Provider
 metadata:
   name: ${aws-provider-name}
 spec:
-  package: xpkg.upbound.io/crossplane-contrib/provider-aws:${coalesce(provider-aws-version, "v0.33.0")}
+  package: xpkg.upbound.io/crossplane-contrib/provider-aws:${coalesce(provider-aws-version, "v0.34.0")}
```

lets remove this - either its passed in or not, but we shouldn't need to manage versions in two places
```suggestion
  package: xpkg.upbound.io/crossplane-contrib/provider-aws:${provider-aws-version}
```

---

thanks, I saw it there was not sure to remove it.